pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -173,55 +178,33 @@ type libInfo struct {
 // extractLibInfo returns the language, the image,
 // and a boolean indicating whether the library should be injected into the pod
 func extractLibInfo(pod *corev1.Pod, containerRegistry string) []libInfo {
-	libInfoList := []libInfo{}
-	podAnnotations := pod.GetAnnotations()
-	for _, lang := range supportedLanguages {
-		customLibAnnotation := strings.ToLower(fmt.Sprintf(customLibAnnotationKeyFormat, lang))
-		if image, found := podAnnotations[customLibAnnotation]; found {
-			libInfoList = append(libInfoList, libInfo{
-				lang:  lang,
-				image: image,
-			})
-		}
+	libInfoMap := map[string]libInfo{}
 
-		libVersionAnnotation := strings.ToLower(fmt.Sprintf(libVersionAnnotationKeyFormat, lang))
-		if version, found := podAnnotations[libVersionAnnotation]; found {
-			image := fmt.Sprintf("%s/dd-lib-%s-init:%s", containerRegistry, lang, version)
-			libInfoList = append(libInfoList, libInfo{
-				lang:  lang,
-				image: image,
-			})
+	// Inject all libraries if Single Step Instrumentation is enabled
```

Comment:
I think the function name needs to change since it doesn't reflect at all that libraries are being injected

Reply:
Do you mean the function name `injectAll`?

Reply:
oh sorry, I meant the parent `extractLibInfo` which is now injecting libraries as well as extracting the library info

Reply:
Renamed `injectAll` to `getAllLibsToInject`